### ISSUE TITLE
aheatmap: Legend text for log scale

### ISSUE THREAD

**Issue by sjackman** [renozao/NMF#49]
I've log scaled the data before plotting, and next to the legend I'd like to display the text "log_10(TPM)" (transcripts per million). Is that possible?

**Comment by renozao** [comment]
Do you mean as a legend title?

**Comment by slowkow** [comment]
It would be wonderful if the title could work like this:

```r
aheatmap(
  ...,
  legendTitle = bquote("log"[10] ~ "TPM")
)
```

And the result would look like this:

![nmf-49](https://cloud.githubusercontent.com/assets/209714/8934473/c714dea0-3515-11e5-9f90-995ac1f7dbec.png)

**Comment by sjackman** [comment]
Yes, precisely, a legend title. If we want to get into fancy formatting as Kamil suggests, I'd love one of
1. LaTeX math `$\log_{10} TPM$`
2. Markdown `log~10~TPM`
3. HTML `log<sub>10</sub>TPM`
These are wishlist items. Plain ASCII would be fine.
